pkg/hub/manager.go

```diff
@@ -167,6 +171,7 @@ func RunControllerManager(ctx context.Context, controllerContext *controllercmd.
 	go addOnFeatureDiscoveryController.Run(ctx, 1)
 	if features.DefaultHubMutableFeatureGate.Enabled(features.DefaultClusterSet) {
```

Comment:
I think we need to update the description of the feature gate also.

Reply:
done in https://github.com/open-cluster-management-io/registration/pull/251/files#diff-2f6213e2bd15bb256be4ed430e664c08612e86384445be797a20d5f56d48d101R32